docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+```
+
+Our instance of the test network was deployed without creating an application channel. However, the test network script creates the system channel when you issue the `./network.sh up` command. Under the covers, the script uses the `configtxgen` tool and the `configtx.yaml` file to build the genesis block of the system channel.  Because the system channel is used to create other channels, we need to take some time to understand the orderer system channel before we can create an application channel.
+
+## The orderer system channel
+
+The first channel that is created in a Fabric network is the system channel. The system channel defines the set of ordering nodes that form the ordering service and the set of organizations that serve as ordering service administrators.
+
+The system channel also includes the organizations that are members of blockchain [consortium](../glossary.html#consortium).
+The consortium is a set of peer organizations that belong to the system channel, but are not administrators of the ordering service. Consortium members have the ability to create new channels and include other consortium organizations as channel members.
+
+The genesis block of the system channel is required to deploy a new ordering service. The test network script already created the system channel genesis block when you issued the `./network.sh up` command. The genesis block was used to deploy the single ordering node, which used the block to create the system channel and form the ordering service of the network. If you examine the output of the `./network.sh` script, you can find the command that created the genesis block in your logs:
+```
+configtxgen -profile TwoOrgsOrdererGenesis -channelID system-channel -outputBlock ./system-genesis-block/genesis.block
+```
+
+The `configtxgen` tool used the `TwoOrgsOrdererGenesis` channel profile from `configtx.yaml` to write the genesis block and store it in the `system-genesis-block` folder. You can see the `TwoOrgsOrdererGenesis` profile below:
+```yaml
+TwoOrgsOrdererGenesis:
+    <<: *ChannelDefaults
+    Orderer:
+        <<: *OrdererDefaults
+        Organizations:
+            - *OrdererOrg
+        Capabilities:
+            <<: *OrdererCapabilities
+    Consortiums:
+        SampleConsortium:
+            Organizations:
+                - *Org1
+                - *Org2
+```
+
+The `Orderer:` section of the profile creates the single node Raft ordering service used by the test network, with the `OrdererOrg` as the ordering service administrator. The `Consortiums` section of the profile creates a consortium of peer organizations named `SampleConsortium:`. Both peer organizations, Org1 and Org2, are members of the consortium. As a result, we can use the test network to create a channel that contains both Org1 and Org2. If we wanted to include another organization in the channel, we would need to add the new organization by [updating the channel configuration](../channel_update_tutorial.html).
```

Comment:
This last sentence makes it sound like the only way to add other orgs to the channel is through the channel update process, when it would be possible to add org3 right here if it was a member of the consortium. So maybe something like:

For information about adding an organization to the channel after the channel has been created, check out [link].